src/lib/__tests__/utils-test.js

```diff
@@ -899,6 +899,38 @@ describe('utils.deprecate', () => {
   });
 });
 
+describe('utils.warn', () => {
+  it('expect to print a warn message', () => {
+    const message = 'message';
+    const warn = jest.spyOn(global.console, 'warn');
+
+    utils.warn(message);
+
+    expect(warn).toHaveBeenCalled();
```

Comment:
Shouldn't we assert that the warn function is called with the right string prefixed by `[InstantSearch.js]: `? (see [`toHaveBeenCalledWith()`](https://jestjs.io/docs/en/expect#tohavebeencalledwitharg1-arg2-))

Reply:
Yes we can, will do it.